test/active_model_serializers/railtie_test_isolated.rb

```diff
@@ -18,6 +18,12 @@ class WithRails < RailtieTest
         "ActionController::Serialization should be included in ActionController::Base, but isn't"
     end
 
+    test 'prepares url_helpers for SerializationContext' do
+      assert ActiveModelSerializers::SerializationContext.url_helpers.respond_to? :url_for
+      assert_equal Rails.application.routes.default_url_options,
+        ActiveModelSerializers::SerializationContext.default_url_options
+    end
```

Comment:
`ActiveModelSerializers::SerializationContext` is an `uninitialized constant` in the non-rails test area. Adding that test seemed to repeat https://github.com/remear/active_model_serializers/blob/url-helpers/test/active_model_serializers/serialization_context_test_isolated.rb#L49-L56. Thoughts?